File: MT.Model/Business/ProjectAcceptanceBasic.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
	#region ProjectAcceptanceBasic:EntityBase
	/// <summary>
	/// This object represents the properties and methods of a ProjectAcceptanceBasic:EntityBase.
	/// </summary>
	public class ProjectAcceptanceBasic:EntityBase
	{
        [Key]
		public Guid ProjectAcceptanceBasicID {get;set;}
		public Guid ProjectID{get;set;}
		public DateTime EstablishedDate{get;set;}
		public DateTime MeetingDate{get;set;}
		public int Status{get;set;}
		public bool Inactive{get;set;}
		public int SortOrder{get;set;}

        /// <summary>
        /// Danh sách file đính kèm
        /// </summary>
        /// Create by: manh:11.02.2018
        public List<ProjectAcceptanceBasic_AttachDetail> Details { get; set; }

    }
	#endregion
}

File: BECore/app/model/ProjectAcceptanceBasic.js
/// <summary>
/// Model EXTJS ProjectAcceptanceBasic.
/// </summary>
Ext.define('QLDT.model.ProjectAcceptanceBasic', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectAcceptanceBasicID',
    idType: 'Guid',
    fields: [
		{ name: "ProjectID", type: "string" },
		{ name: "EstablishedDate", type: "date" },
		{ name: "MeetingDate", type: "date" },
		{ name: "Status", type: "int" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});

